spacy_llm/tasks/__init__.py

```diff
@@ -15,7 +16,9 @@
     "make_spancat_task",
     "make_spancat_task_v2",
     "make_textcat_task",
+    "make_lemma_task",
     "TextCatTask",
     "SpanCatTask",
     "RELTask",
+    "LemmaTask",
```

Comment:
It looks like this list has duplicates in it (e.g. `TextCatTask`) - probably due to merge conflicts on a previous PR. Let's have this entirely alphabetized so it's easier to maintain.

Reply:
Fixed in https://github.com/explosion/spacy-llm/pull/164/commits/01790a0148f0a87c5594e9735f97de7f6b1620d6.